variables.tf

```diff
@@ -1,12 +1,16 @@
 variable "arguments" {
-  type = "list"
+  default = [""]
+  type    = "list"
 }
 
 variable "envs" {
-  type = "list"
+  default = [""]
+  type    = "list"
 }
 
-variable "playbook" {}
+variable "playbook" {
+  default = ""
```

Comment:
I assume, empty default value will cause an error.
`playbook` path is required.

Reply:
@comeanother, look at `  count = "${signum(length(var.playbook)) == 1 ? 1 : 0}"
`